src/components/calcite-button/calcite-button.scss

```diff
@@ -619,3 +619,18 @@
 :host([scale="l"][icon-start][icon-end][appearance="transparent"]) a:not(.content--slotted) {
   --calcite-button-padding-x: theme("padding.11");
 }
+
+// accomodate color changes when button inside inline-editable with controls 
+:host(.calcite-inline-editable__cancel-editing-button[appearance=transparent][color=neutral]) {
+  button:not(.content--slotted),
+  a:not(.content--slotted) {
+    // color: red;
```

Comment:
Can you remove the commented out code and add quotation marks for the attribute selectors?

Since latest button colors have just been merged, I think we can move this selector to the other transparent/neutral group on [line 604](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-button/calcite-button.scss#L604) (thereby fixing my "loadder" typo and only target the button):
```scss
:host([appearance="transparent"][color="neutral"]:not(.calcite-inline-editable__cancel-editing-button)) {
  button,
  a,
  calcite-loader {
    @apply text-color-1;
  }
}

:host([appearance="transparent"][color="neutral"].calcite-inline-editable__cancel-editing-button) {
  button {
    @apply text-color-3;
  
    &:hover {
      @apply text-color-1;
    }
  }
}
```

Reply:
Sounds good. I will merge in master and get my code updated to line this up. Thanks.